engine/runtime/src/test/scala/org/enso/compiler/test/pass/analyse/DataflowAnalysisTest.scala

```diff
@@ -1286,26 +1287,27 @@ class DataflowAnalysisTest extends CompilerTest {
       dependencies.getDirect(vecId) shouldEqual Some(Set(xUseId, yId, litId))
     }
 
-    "work properly for typeset literals" ignore {
-      // FIXME: Not supported by new parser--needs triage (#5894).
+    "work properly for typeset literals" in {
       implicit val inlineContext: InlineContext = mkInlineContext
 
       val ir =
         """
           |{ x := a ; y := b }
           |""".stripMargin.preprocessExpression.get.analyse
 
-      val depInfo = ir.getMetadata(DataflowAnalysis).get
+      if (!ir.isInstanceOf[IR.Error.Syntax]) {
```

Comment:
Similarly as in the previous case, if it is reported as a syntax error, we should fail the test?

Reply:
This test is testing _"map literals"_ (it is a surprising revelation to realize they were actually supported by the old parser). Right now this syntax yields a `IR.Error.Syntax`, however the like a hood _"map literals"_ are again supported is high (very likely with the same syntax).

The test is modified to pass when there is an error and to "be evaluated again" as soon as the syntax gets recognized again.